src/intl/az/page-layer-2.json

```
@@ -18,7 +18,7 @@
   "layer-2-what-is-layer-1-title": "Qat 1 nədir?",
   "layer-2-what-is-layer-1-1": "Qat 1 əsas blokçeyndir. Ethereum və Bitcoin hər ikisi qat 1 blokçeynləridir, çünki onlar<strong>müxtəlif qat 2 şəbəkələrin onun üzərində qurulduğu əsas təməldir</strong>. Qat 2 layihələrinin nümunələri arasında Ethereum-da \"toplamalar\" və Bitcoin-da Lightning Network daxildir. Bu qat 2 layihələrində bütün istifadəçi əməliyyatları son nəticədə 1-ci qat blokçeyninə geri qayıda bilər.",
   "layer-2-what-is-layer-1-2": "Ethereum həmçinin qat 2-lər üçün məlumatların mövcudluğu qatı kimi fəaliyyət göstərir. Qat 2 layihələri, məlumatların mövcudluğu üçün Ethereum-a güvənərək, əməliyyat məlumatlarını Ethereum-da yerləşdirəcək. Bu məlumatlar qat 2-nin vəziyyətini öyrənmək və ya qat 2 üzrə əməliyyatları mübahisə etmək üçün istifadə edilə bilər.",
-  "layer-2-what-is-layer-1-list-title": "<strong>Qat 1 kimi Ethereum-da daxildir:</strong>",
+  "layer-2-what-is-layer-1-list-title": "<strong>Qat 1 kimi Etheruem-da daxildir:</strong>",
```

```suggestion
  "layer-2-what-is-layer-1-list-title": "<strong>Qat 1 kimi Ethereum-da daxildir:</strong>",
```

416 uses of "Ethereum" for `az` files so I'm going to assume this was a mistake.